README.md

```diff
@@ -1,710 +1,81 @@
-By default, you will be in Hiro's development branch, `develop`, with code that has not been released yet. If you plan on submitting any changes to the code, then this is the right branch for you.
-If you want the latest stable version, switch to the main branch by entering the following command:
-
-```bash
-git checkout main
-```
-
-## Getting started with Clarinet
-
-The following sections describe how to create a new project in Clarinet and populate it with smart contracts. Clarinet
-also provides tools for interacting with your contracts in a REPL, and performing automated testing of contracts.
-
-### Setup shell completions
-
-Clarinet has many different built-in commands, which are useful to enable tab-completion in your shell. You can use `clarinet` to generate the shell completion scripts for many common shells using the command:
-
-```sh
-clarinet completions (bash|elvish|fish|powershell|zsh)
-```
-
-After generating the file, please refer to the documentation for your shell to determine where this file should be moved and what other steps may be necessary to enable tab-completion for `clarinet`.
-
-### Create a new project
-
-Once you have installed Clarinet, you can create a new project by entering the following command:
-
-```bash
-clarinet new my-project && cd my-project
-```
-
-Clarinet will create a project directory with the following directory layout:
-
-```bash
-.
-├── Clarinet.toml
-├── contracts
-├── settings
-│   └── Devnet.toml
-│   └── Testnet.toml
-│   └── Mainnet.toml
-└── tests
-```
-
-The `Clarinet.toml` file contains configuration for the smart contracts in your project. When you create contracts in
-your project, Clarinet will add them to this file.
-
-The `settings/Devnet.toml` file contains configuration for accounts in the Clarinet console, including the seed
-phrases and initial balances. Initial balances are in microSTX.
-
-For a detailed video description on how you can create a new project, please see the [Creating a New Project](https://www.youtube.com/watch?v=F_Sb0sNafEg&list=PL5Ujm489LoJaAz9kUJm8lYUWdGJ2AnQTb&index=4) YouTube video.
+---
 
-### Add a new contract
+### Documentation
 
-Clarinet can handle adding a new contract and its configuration to your project with the following command:
+- [Clarinet →](https://docs.hiro.so/stacks/clarinet)  
+- [Clarinet JS SDK →](https://docs.hiro.so/stacks/clarinet-js-sdk)
 
-```bash
-clarinet contract new bbtc
-```
+---
 
-Clarinet will add 2 files to your project: the contract file in the `contracts` directory, and the contract test file
-in the `tests` directory.
+### Quickstart
 
 ```bash
-.
-├── Clarinet.toml
-├── contracts
-│   └── bbtc.clar
-├── settings
-│   └── Devnet.toml
-│   └── Mainnet.toml
-│   └── Testnet.toml
-└── tests
-    └── bbtc_test.ts
-```
-
-Clarinet will also add configuration to the `Clarinet.toml` file for your contract.
-
-```toml
-[project.cache_location]
-path = ".cache"
-[contracts.bbtc]
-path = "contracts/bbtc.clar"
+# Install Clarinet
```

Comment:
Could be nice to link to the doc for other installation method

Reply:
yeah i added a callout underneath this section that links out to our installation section in docs, good call